Failed to save texture

### Minecraft Version

1.21.1

### KubeJS Version

2101.7.2-build.295

### Rhino Version

2101.2.7-build.81

### Architectury Version

13.0.8

### Forge/Fabric Version

NeoForge 21.1.209

### Describe your issue

startup_scripts/main.js
```javascript
StartupEvents.registry("fluid", (event) => {
  event.create("test_fluid");
});
```

<img width="858" height="527" alt="Image" src="https://github.com/user-attachments/assets/90d6ca38-3618-4bd1-ba04-36277468f523" />

client.log
```
[22:59:48] [INIT] KubeJS 2101.7.2-build.295; MC 2101 NeoForge
[22:59:48] [INIT] Loaded plugins:
[22:59:48] [INIT] - dev.latvian.mods.kubejs.plugin.builtin.BuiltinKubeJSPlugin
[22:59:48] [INIT] - dev.latvian.mods.kubejs.plugin.builtin.BuiltinKubeJSClientPlugin
[22:59:48] [INIT] - dev.latvian.mods.kubejs.integration.architectury.ArchitecturyIntegration
[22:59:48] [INIT] - com.jesz.createdieselgenerators.compat.kubejs.CDGKubeJSPlugin
[22:59:48] [INIT] - com.tom.createores.kubejs.KubeJSExcavation
[22:59:48] [INIT] - moe.wolfgirl.probejs.plugin.BuiltinProbeJSPlugin
[22:59:48] [INIT] - moe.wolfgirl.probejs.plugin.ProbeJSKJSPlugin
[22:59:48] [INIT] - dev.ftb.mods.ftbxmodcompat.neoforge.ftbquests.kubejs.FTBQuestsKubeJSPlugin
[22:59:48] [INIT] - dev.ftb.mods.ftbxmodcompat.neoforge.ftbteams.kubejs.FTBTeamsKubeJSPlugin
[22:59:48] [INFO] client_scripts:main.js#2: Hello, World! (Loaded client example script)
[22:59:48] [INFO] Loaded script client_scripts:main.js in 0.002 s
[22:59:48] [INFO] Loaded 1/1 KubeJS client scripts in 0.033 s with 0 errors and 0 warnings
[22:59:59] [ERROR] ! Failed to save texture kubejs:block/generated/test_fluid_still
[22:59:59] [ERROR] ! Failed to save texture kubejs:block/generated/test_fluid_flow
[22:59:59] [INFO] Client resource reload complete!
```

### Crash report/logs

_No response_

--- a/src/main/java/dev/latvian/mods/kubejs/fluid/FluidBuilder.java
+++ b/src/main/java/dev/latvian/mods/kubejs/fluid/FluidBuilder.java
@@ -160,13 +160,13 @@ public class FluidBuilder extends BuilderBase<FlowingFluid> {
 	public void generateAssets(KubeAssetGenerator generator) {
 		var stillTexture = generator.loadTexture(fluidType.stillTexture);
 
-		if (stillTexture != LoadedTexture.EMPTY) {
+		if (!(stillTexture.width <= 0 || stillTexture.height <= 0)) {
 			generator.texture(fluidType.actualStillTexture, stillTexture.tint(fluidType.tint));
 		}
 
 		var flowingTexture = generator.loadTexture(fluidType.flowingTexture);
 
-		if (flowingTexture != LoadedTexture.EMPTY) {
+		if (!(stillTexture.width <= 0 || stillTexture.height <= 0)) {
 			generator.texture(fluidType.actualFlowingTexture, flowingTexture.tint(fluidType.tint));
 		}
 

